sycl/test-e2e/VirtualMem/memory_granularity.cpp

```diff
@@ -0,0 +1,48 @@
+// This test checks whether memory granularity returned is greater than 0 and
+// that the recommended granularity is the same size or greater than the minimum
+// granularity for virtual memory extension.
+
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/detail/core.hpp>
+
+#include <sycl/ext/oneapi/virtual_mem/virtual_mem.hpp>
+
+#include <assert.h>
```

Comment:
Most sycl-e2e tests go for `cassert` instead of `assert.h`. I think we should align.

Reply:
Sure